hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/HoodieFlinkWriteClient.java

```diff
@@ -270,51 +254,8 @@ public void preWrite(String instantTime, WriteOperationType writeOperationType,
     // remove the async cleaning
   }
 
-  @Override
-  protected void writeTableMetadata(HoodieTable table, String instantTime, String actionType, HoodieCommitMetadata metadata) {
```

Comment:
please annotate in comments properly to explain the code removals

Reply:
Already update.